owner/fail_test.go

```diff
@@ -63,6 +63,10 @@ var (
 )
 
 func (s *testSuite) TestFailNewSession(c *C) {
+	// New version of clientv3 can not test use this unix socket.
```

Comment:
Is it possible that we use [api_gate_way](https://github.com/etcd-io/etcd/blob/master/Documentation/dev-guide/api_grpc_gateway.md) to replace it?